cql3/statements/create_index_statement.cc

```diff
@@ -67,13 +68,13 @@ create_index_statement::validate(query_processor& qp, const service::client_stat
     if (_raw_targets.empty() && !_properties->is_custom) {
         throw exceptions::invalid_request_exception("Only CUSTOM indexes can be created without specifying a target column");
     }
-
+    
```

Comment:
Please don't make gratuitous whitespace changes, especially not ones that add trailing whitespace.